pkgs/sketch_pad/lib/main.dart

```diff
@@ -143,7 +143,7 @@ class _DartPadAppState extends State<DartPadApp> {
             ColorScheme.fromSeed(seedColor: lightPrimaryColor).copyWith(
           surface: lightSurfaceColor,
           onSurface: Colors.black,
-          surfaceVariant: lightSurfaceVariantColor,
+          surfaceContainerHighest: Colors.blue,
```

Comment:
Why this change (and the similar one below)?

Reply:
I was getting an error in pipeline it was showing an error , which was exactly like this :- 

 info • 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre • lib/main.dart:146:11 • deprecated_member_use
   info • 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre • lib/main.dart:167:11 • deprecated_member_use
   info • 'surfaceVariant' is deprecated and shouldn't be used. Use surfaceContainerHighest instead. This feature was deprecated after v3.18.0-0.1.pre • lib/problems.dart:46:30 •

Reply:
I thought may be changing it would resolve the error

Reply:
It looks like we hit this in another PR as well: https://github.com/dart-lang/dart-pad/pull/2870/files#diff-ed60136a3d9ee7c71a9ed356332807808330f0cb99e3959050cfe26b087d6f8eR146. You could wait until that lands and then update your branch w/ the contents of `main`.